src/game/title_screen.c

```diff
@@ -1472,7 +1472,7 @@ static void WavesBackgroundAnim(TitleScreen *titleScreen)
 
     gFlags |= FLAGS_EXECUTE_HBLANK_CALLBACKS;
     gFlags |= FLAGS_EXECUTE_HBLANK_COPY;
-    gHBlankCopySize = 16;
+    gHBlankCopySize = sizeof(BgAffineReg);
```

Comment:
Nice, I wonder how this was working before? Or do the affine regs not change size for the widescreen hack?

Reply:
Nah, they don't! ^^